about.html

```diff
@@ -0,0 +1,324 @@
+    <title>Nigeria Music Concert NMC 2023</title>
+    <link rel="icon" type="image/x-icon"
+        href="https://img.freepik.com/premium-vector/retro-microphone-vintage-background-poster_606304-137.jpg?w=2000">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="css/main.css" />
+</head>
+
+<body>
+    <div class="container-fluid">
+        <!--upper navigation-->
+        <nav>
+            <div class="row top_navigation">
+                <div class="col-9"></div>
+                <div class="col-3">
+                    <ul>
+                        <li>
+                            <img src="images/facebook.png" alt="facebook">
+                        </li>
+                        <li>
+                            <img src="images/twitter.png" alt="twitter">
+                        </li>
+                        <li>English</li>
+                        <li>My page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <div class="col-2"></div>
+            </div>
+        </nav>
+
+        <!--Lower navigation-->
+        <nav>
+            <div class="row lower_navigation">
+                <div class="col-1"></div>
+                <div class="col-2 about_logo">
+                    <img src="images/nmc 2.webp" alt="">
+                </div>
+                <div class="col-1"></div>
+                <div class="col-8">
+                    <ul>
+                        <li>
+                            <a href="index.html">Home</a>
+                        </li>
+                        <li>
+                            <a href="##">Program</a>
+                        </li>
+                        <li>
+                            <a href="##">Join</a>
+                        </li>
+                        <li>
+                            <a href="##">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="##">News</a>
+                        </li>
+                        <li>
+                            <a href="##">Follow us</a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+        </nav>
+
+        <!--main section-->
+        <div class="container-fluid main-section"
+            style="background-image:url('./images/bg 2.jpg');background-repeat:no-repeat;background-size:cover;margin:0px 0px !important;">
+
+            <div class="container hambuger" id="hambuger_about">
+                <div class="row">
+                    <div class="col-1 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
```

Comment:
- [ ] Dear @fauziyahzeeyah, I like your layout and I honestly like the way you customize your page. But its among the main requirement to stick to the given design `The pages should look almost identical to the original design, outside the content differences`. Please update your `home` and `About` section contents to look the same with the design. this includes `colors for texts`, `backgrounds (background-image)`, `borders`. 

### For your information 
> If you have a problem finding the background image for the giving design please use something similar. 
> Basic color swatch: #272A31, #EC5242, #D3D3D3
> Fonts used originally: Cocogoose, Lato
### Design home page
![Screenshot 2023-01-14 163650](https://user-images.githubusercontent.com/69077061/212474631-b5c85edc-26e2-40c2-bb52-9b719535d5f2.png)
### Your home page
![Screenshot 2023-01-14 163630](https://user-images.githubusercontent.com/69077061/212474661-a4fb1526-5000-40e8-8081-f7b198244f04.png)
---
### Design about page
![Screenshot 2023-01-14 164028](https://user-images.githubusercontent.com/69077061/212474800-6c78367a-8522-4247-a24f-4dc333dbf16b.png)
### Your about page
![Screenshot 2023-01-14 164058](https://user-images.githubusercontent.com/69077061/212474808-53045c58-5bb5-4b1f-b036-20e24b46c00c.png)